vidplayer/player.go

```diff
@@ -29,6 +29,20 @@ var ErrRTMP = errors.New("ErrRTMP")
 var ErrHLS = errors.New("ErrHLS")
 var PlaylistWaittime = 6 * time.Second
 
+// vars to tag http response headers w/ debugging info 
+var (
+	lpVersion string 
+	nodeType string
+	nodeEthAddr string 
+)
+
+type VidPlayerDebug struct {
+	DebugEnabled	bool
+	LivepeerVersion string
+	NodeType	string
```

Comment:
TODO: remove `NodeType` here as we'll always be looking at the broadcaster node for async HLS playback.